project/plugins.sbt

```diff
@@ -7,3 +7,4 @@ addSbtPlugin("com.alejandrohdezma"       % "sbt-github"               % "0.8.2")
 addSbtPlugin("com.alejandrohdezma"       % "sbt-github-header"        % "0.8.2")
 addSbtPlugin("com.alejandrohdezma"       % "sbt-github-mdoc"          % "0.8.2")
 addSbtPlugin("com.alejandrohdezma"       % "sbt-remove-test-from-pom" % "0.1.0")
+addSbtPlugin("org.lyranthe.sbt"          % "partial-unification"      % "1.1.2")
```

Comment:
ah yep you're right.  I initially added it because I still couldn't get my compiler working and I thought this was the problem but then I realized it was because I was trying to call `sequence` on an instance of `Seq`, not `List`....